src/firebase/api/gcp-utils.js

```diff
@@ -17,7 +17,7 @@ export const uploadImageToStorage = async (
   const storageRef = storage.ref();
   const rawImageBucketRef = storageRef.child(bucket);
   const newImageRef = rawImageBucketRef.child(imagePath);
-  newImageRef.put(image).then((snapshot) => {
+  return newImageRef.put(image).then((snapshot) => {
```

Comment:
I believe this file is still undergoing some development work from the BE.

Reply:
Update: there's a PR open to update the `gcp-utils.js` file, once that's merged, I can update the template.